crates/compiler/test_syntax/tests/snapshots/pass/ann_where_e_newline_implements.expr.formatted.roc

```diff
@@ -1,4 +1,5 @@
-J : [
-] where e
-implements T
+J :
+    [
```

Comment:
Do we want to maintain superfluous newlines inside of empty collections?

Reply:
That's something that I'd like to propose changing at some point, yes - but that's ideally a separate PR